src/polymer-project.ts

```diff
@@ -327,6 +327,24 @@ class HtmlSplitter extends Transform {
           this.push(scriptFile);
         }
 
+        for (let i = 0; i < styleTags.length; i++) {
+            let styleTag = styleTags[i];
+            let source = dom5.getTextContent(styleTag);
+            let extension = 'css';
+            let childFilename = `${osPath.basename(filePath)}_style_${i}.${extension}`;
+            let childPath = osPath.join(osPath.dirname(filePath), childFilename);
+            styleTag.childNodes = [];
+            dom5.setAttribute(styleTag, 'src', childFilename);
```

Comment:
I'm not saying `<style src=>` isn't wrong. Just pointing out that `<link include=>` also isn't valid.  If split without rejoin on a polymer project is to be supported, more needs to be done here to split these styles. Any ideas for how to handle that?